Apps/2FAuth/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 name: 2fauth
 services:
   2fauth:
-    image: 2fauth/2fauth:3.4.2
+    image: 2fauth/2fauth:4.0.2
```

Comment:
That's a change in major version. Have you tested this version against existing configurations in the compose yaml (i.e. env, memory, etc.)?

If not, it's simpler to just update its minor/build version for now.